README.md

```diff
@@ -108,16 +108,44 @@ skew may result in `system time has changed` messages in VM logs.
 #### Network
 
 The guest agent uses network interface metadata to manage the network
-interfaces in the guest by performing the following tasks:
-
-*   Enabled all associated network interfaces on boot.
+interfaces by performing the following tasks:
+
+*   Enable all associated network interfaces on boot.
+    *   Detect the current active network manager service that is managing the
+        primary NIC.
+    *   Rollback and delete any files/configurations left behind by all other
+        supported network manager services.
+    *   Write and apply new configurations for the secondary NICs using the
+        network manager service detected in the first step.
 *   Setup or remove IP routes in the guest for IP forwarding and IP aliases
     *   Only IPv4 IP addresses are currently supported.
     *   Routes are set on the primary ethernet interface.
     *   Google routes are configured, by default, with the routing protocol ID
         `66`. This ID is a namespace for daemon configured IP addresses. It can
         be changed with the config file, see below.
 
+On Linux, supported network managers are (listed by priority):
```

Comment:
should we also document the paths and priority it uses to make it easier to read. 

I am thinking of taking an example of `eth0` as interface and writing down where to expect config files in each case. 

For ex. in case of netplan we write `/run/netplan/20-google-guest-agent-ethernet.yaml` it will be good to have explicit paths here rather than having to go through code to figure out

Reply:
Done. Added the paths and priorities.